src/app/ui/components/containers/dialog/dialog.tsx

```diff
@@ -14,7 +14,7 @@ export interface DialogProps {
 interface RadixDialogProps extends DialogProps {
   children: ReactNode;
   footer?: ReactNode;
-  header?: ReactElement<any, string | JSXElementConstructor<any>>;
+  header: ReactElement<any, string | JSXElementConstructor<any>>;
```

Comment:
Not a change in this PR but, why not ReactNode here 🤔

Reply:
I will change it separately👍 . Not sure why I didn't do that in the first place